src/mezzanine.js

```diff
@@ -0,0 +1,13 @@
+export class Mezzanine {
+    constructor () {
+        this.id = null;
+        this.fileURL = null;
+        this.deliveryType = 'progressive';
+        this.codec = null;
+        this.mimeType = null;
```

Comment:
Shouldn't we call it `type` as in the spec ? and just comment it with this comments in the specs ?
<img width="570" alt="Capture d’écran 2019-08-05 à 17 32 17" src="https://user-images.githubusercontent.com/1483549/62476771-ee7b7300-b7a7-11e9-8c64-ff22d59e69d0.png">
And for all fields shouldn't we put possible value in comment  ?

Reply:
And maybe put null on all mandatory fields and put the default values for the others ?
Just to distinct which one is mandotary and which on is optional WDYT ?